linera-views/README.md

```diff
@@ -11,6 +11,134 @@ atomically persisted in storage).
 
 See the [CONTRIBUTING](../CONTRIBUTING.md) file for how to help out.
 
+## Design
+
+# Possible clients
+
+We designed a `KeyValueStoreClient` trait that implements the basic functionalities
+of a ke-value store whose keys are `Vec<u8>` and whose values are `Vec<u8>`.
```

Comment:
typo: key-value

Reply:
Yes sorry the Grammarly corrections were not upstreamed by accident.